src/Workspaces/Core/Portable/Workspace/Solution/DocumentState_TreeTextSource.cs

```diff
@@ -60,6 +60,9 @@ public bool TryGetTextVersion(LoadTextOptions options, out VersionStamp version)
                 version = _version;
                 return version != default;
             }
+
+            public Task<VersionStamp> GetVersionAsync(LoadTextOptions options, CancellationToken cancellationToken)
+                => Task.FromResult(_version);
```

Comment:
❔ How often is this called? Should we change `_version` from `VersionStamp` to `Task<VersionStamp>`?

Reply:
the expected case for all of this, once initial load is done is that this will almost allways be synchronous.  So i'm going to change to ValueTask for all of these.

Reply:
done.